lib/executable-config/lookup/obelisk-executable-config.cabal

```diff
@@ -31,3 +31,5 @@ library
     filepath,
     text,
     transformers
+
+  ghc-options: -Wall
```

Comment:
If we ever want to put this on Hackage we'll have to remove this again, because Hackage. I am not personally against having it in Obelisk tho ...

Reply:
It's in all the other Obelisk libraries.